runtime/src/parameters.rs

```diff
@@ -63,6 +97,15 @@ parameter_types! {
   pub const OperationalFeeMultiplier: u8 = 5;
   pub const TransactionByteFee: Balance = 100 * MICRO;
 
-  // XCM
-  pub const MaxInstructions: u32 = 100;
+  // Treasury
+  pub const Burn: Permill = Permill::from_percent(50);
+  pub const MaxApprovals: u32 = 100;
+  pub const ProposalBond: Permill = Permill::from_percent(5);
+  pub const ProposalBondMinimum: Balance = 10 * BASE;
```

Comment:
I think this is fine, since when a proposal is rejected the complete proposal bond will be slashed, even if in theory that proposal made sense. [Moonbeam](https://github.com/PureStake/moonbeam/blob/617f917b236e93b807daba6086e2823d8485893f/runtime/moonbeam/src/lib.rs#L554), [Acala](https://github.com/AcalaNetwork/Acala/blob/5a8f381a95649eb74385fb7db31421d78118a651/runtime/acala/src/lib.rs#L568) and [Kusama](https://github.com/paritytech/polkadot/blob/22967a4128702b12e438bb951aae32927b949dcd/runtime/kusama/src/lib.rs#L737) use 1, 10 and 20 native currency respectively. I suggest we start of with ten and increase it in the future in case it is insufficient.